src/toHex.ts

```diff
@@ -6,7 +6,18 @@ import guard from './guard';
  */
 function toHex(color: string): string {
   const [r, g, b, a] = parseToRgba(color);
-  const hex = (x: number) => guard(0, 255, x).toString(16).padStart(2, '0');
+
+  let hex = (x: number) => {
+    const h = guard(0, 255, x).toString(16).padStart(2, '0');
```

Comment:
sneaky `padStart` is still in there haha

Reply:
lol whoops, one sec